processor/src/main/java/org/mapstruct/ap/processor/DefaultVersionInformation.java

```diff
@@ -0,0 +1,168 @@
+/**
+ *  Copyright 2012-2015 Gunnar Morling (http://www.gunnarmorling.de/)
+ *  and/or other contributors as indicated by the @authors tag. See the
+ *  copyright.txt file in the distribution for a full listing of all
+ *  contributors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.mapstruct.ap.processor;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.jar.Manifest;
+
+import javax.annotation.processing.ProcessingEnvironment;
+
+import org.mapstruct.ap.version.VersionInformation;
+
+/**
+ * Provides information about the processor version and the processor context implementation version
+ *
+ * @author Andreas Gudian
+ */
+public class DefaultVersionInformation implements VersionInformation {
```

Comment:
Funny, somehow Sjaak always finds another link to the changed files - he commented in the same direction, and my answer was: to avoid cyclic package dependencies (as I hand in the version information to the model). I'll add a comment.